README.md

```diff
@@ -149,3 +163,35 @@ Populating an index is done by pushing score items to a queue.
 where `${cmd}` is the command to run, e.g. `dotnet osu.ElasticIndexer.dll queue`
 
 # Typical use-cases
+
+## Queuing items for processing from another client
+
+Push items into the Redis queue "`osu-queue:score-index-${schema}`"
+e.g.
+
+```csharp
+ListLeftPush("osu-queue:score-index-1", "{ \"Action\": \"index\",\"ScoreId\": 1 }");
+```
+
+or from redis-cli:
+```
+LPUSH "osu-queue:score-index-1" "{\"Action\":\"index\",\"ScoreId\":1}"
+```
+
+### Indexing a score by `id`
+```json
+{ "Action": "index", "ScoreId": 1 }
+```
+
+### Deleting a score by `id`
+```json
+{ "Action": "delete", "ScoreId": 1 }
+```
```

Comment:
What's the case for deleting? Wondering if we even need an action if the deletion case is always "doesn't exist in database or has `preserve=0`.

Reply:
So a document can be deleted directly when testing for effects.

Reply:
So it will never be used in actual usages?

Reply:
No, it's a convenience when testing

Reply:
I mean, considering it's going to be fetching from a mysql table at the end of the day (where you can set `preserve:false`) it's probably fine to remove here to keep things simple then?

Reply:
Being able to just delete it makes it easier than needing to flip `preserve` between true and false on he db first; with the command I can just do `score delete 1` to remove and `score index 1` to put it back; it also means not needing an entire infrastructure and complete dataset to always be present and be in sync when testing 👀